docs/special-tricks.md

```diff
@@ -0,0 +1,29 @@
+# Special tricks
+
+This is a file in which we can collect some tricks that we might otherwise just put in private note files.
+We have a file to share these in instead here.
+
+### Changing the API in the web app
+
+In the browser console, run `window.localStorage.setItem('api-url', 'https://cms-test.integreat-app.de')`
+or whichever API (e.g. ''https://cms.integreat-app.de) you want to call. You can also do it manually in
+the local storage of the browser.
```

Comment:
This is a duplicate (web/README.md#cms)

Reply:
It may be a duplicate but both Steffi and I had it in our notes so it's clearly not visible. I also don't really see this information changing so I don't think it's a problem that it's written down in two places.